MaxText/decode.py

```diff
@@ -173,6 +174,8 @@ def prefill_or_load(config, model, state, rng, sp_tokenizer, mesh, state_mesh_sh
 
 def decode_loop(config, state=None):
   """Decoding loop for the Transformer model."""
+  assert config.add_eos is False,\
+    "For decoding, we set eos to be False"
```

Comment:
"For decoding, we must set add_eos=False"

Reply:
done